esphome/components/nfc/ndef_message.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "esphome/core/log.h"
+#include "ndef_record.h"
+
+static const uint8_t MAX_NDEF_RECORDS = 4;
+
+namespace esphome {
+namespace nfc {
+
+class NdefMessage {
+ public:
+  NdefMessage(){};
+  NdefMessage(std::vector<uint8_t> data);
+
+  std::vector<NdefRecord> get_records() { return this->records_; };
+
+  boolean add_record(NdefRecord* record);
```

Comment:
```suggestion
  bool add_record(NdefRecord* record);
```

I know Arduino.h exposes that macro, but we should be using the C++ keywords instead.